.buildkite/nightly-benchmarks/latency-tests.json

```diff
@@ -0,0 +1,34 @@
+[
+    {
+        "test_name": "latency_llama8B_tp1",
```

Comment:
QQ: is it possible to choose hardwares here? Or is it just using L4?

Reply:
No, the current CI machine is 8xA100. Our current plan is to cover both A100 and H100 in the future, but we will consider other hardwares if we have sufficient resources for CI.

Reply:
sounds good we start from A100. Can you update this to README.md?

Reply:
sure sure